redirects-list.json

```diff
@@ -47,10 +47,12 @@
   "^/doc/start/sharing-data-and-model-files$                                              /doc/start/data-and-model-versioning#storing-and-sharing 302",
   "^/doc/tutorial(/.*)?$                                                                  /doc/start",
   "^/doc/tutorials/get-started(/.*)?$                                                     /doc/start",
-  "^/doc/tutorials/versioning(/.*)?$                                                      /doc/use-cases/versioning-data-and-model-files/tutorial",
+  "^/doc/tutorials/versioning(/.*)?$                                                      /doc/use-cases/versioning-data-and-models/tutorial",
+  "^/doc/use-cases/versioning-data-and-model-files/tutorial$                              /doc/use-cases/versioning-data-and-models/tutorial",
   "^/doc/tutorials(/.*)?                                                                  /doc/start",
 
-  "^/doc/use-cases/data-and-model-files-versioning/?$                                     /doc/use-cases/versioning-data-and-model-files",
+  "^/doc/use-cases/data-and-model-files-versioning/?$                                     /doc/use-cases/versioning-data-and-models",
+  "^/doc/use-cases/versioning-data-and-model-files/?$                                     /doc/use-cases/versioning-data-and-models",
```

Comment:
This is part of the Versioning Data case URL fix. I'm updating existing redirects to final locations per https://github.com/iterative/dvc.org/pull/3772#discussion_r924066774.